rootfs/rootfs/etc/rc.d/vmtoolsd

```diff
@@ -1,14 +1,18 @@
 #!/bin/sh
 
-# Creates mountpoint for Shared Folders
-if [ ! -d /mnt/hgfs ]; then
-	mkdir -p /mnt/hgfs
-	# load shared folders kernel module
-	modprobe vmhgfs
-	# try to mount the root shared folder, this command can fail
-	# if shared folders are disabled on the host, vmtoolsd will take care of the mount
-	# if they are enabled while the machine is running.
-	mount -t vmhgfs .host:/ /mnt/hgfs
-fi
+SYSTYPE=$(cat /sys/class/dmi/id/sys_vendor | grep -i vmware | wc -l)
```

Comment:
"ic" :+1: (sorry, I definitely didn't plan that pun the second I typed out
my original comment...)